test/slicing/test_slice_combiner.py

```diff
@@ -253,6 +252,8 @@ def test_temperature(self):
 
         # Check number of elements that match either of the original weights
         num_matching_original = torch.sum(
-            torch.isclose(combined_rep, torch.ones(batch_size, h_dim) * 2)
-        ) + torch.sum(torch.isclose(combined_rep, torch.ones(batch_size, h_dim) * 4))
+            torch.isclose(combined_rep, torch.ones(batch_size, h_dim) * 2, atol=1e-4)
+        ) + torch.sum(
```

Comment:
nit: hard to read on multiline. can we name the two torch.sum ouputs something meaningful?